gradio/interface.py

```diff
@@ -107,7 +107,7 @@ def __init__(
         examples_per_page: int = 10,
         example_labels: list[str] | None = None,
         live: bool = False,
-        title: str | None = None,
+        title: str = "Gradio",
```

Comment:
Same, we shouldn't change this b/c the `title` is also printed as Markdown on top of the Interface UI